SoftLayer/CLI/virt/upgrade.py

```diff
@@ -20,15 +22,17 @@
               help="CPU core will be on a dedicated host server.")
 @click.option('--memory', type=virt.MEM_TYPE, help="Memory in megabytes")
 @click.option('--network', type=click.INT, help="Network port speed in Mbps")
+@click.option('--add', type=click.INT, required=False, help="add Hard disk in GB")
```

Comment:
Change this to `--add-disk` and have it use multiple so they can add more than 1 disk.